File: 프로그래머스/unrated/181927. 마지막 두 원소/마지막 두 원소.py
def solution(num_list):
    answer = []
    for n in num_list:
        answer.append(n)
    if (num_list[-1] > num_list[-2]):
        answer.append(num_list[-1]-num_list[-2])
    else:
        answer.append(num_list[-1]*2)
    return answer

File: 프로그래머스/unrated/181927. 마지막 두 원소/마지막 두 원소.swift
import Foundation

func solution(_ num_list:[Int]) -> [Int] {
    var result = num_list
    if (num_list[result.count-1] > num_list[result.count-2]) {
        result.append(num_list[result.count-1] - num_list[result.count-2])
    } else {
        result.append(num_list[result.count-1]*2)
    }
    return result
}